app/main/views/g_cloud.py

```diff
@@ -574,7 +574,9 @@ def save_search(framework_family):
                                errors=get_errors_from_wtform(form),
                                form=form,
                                save_search_options=save_search_options,
-                               search_summary_sentence=search_summary.markup(),
+                               search_summary_sentence=Markup(
+                                   f'<p class="app-search-summary govuk-body-s">{search_summary.markup()}</p>'
```

Comment:
since we share this wrapper tag in both usages could we move it into a shared helper or have `search_summary` include it?

Reply:
Done, by adding a `wrap` arg to `SearchSummary.markup()`